modules/php/States/DrawCardsTrait.php

```diff
@@ -30,18 +30,24 @@ public function st_drawCards()
       RuleInflation::notifyActiveFor($player_id);
     }
     $partyBonus =
-      Utils::getActivePartyBonus() && Utils::isPartyInPlay()
+      Utils::getActivePartyBonus()
```

Comment:
Is there a way to write this that would reuse `recheckForPartyBonus` and the other helpers you created?

Reply:
Yeah, that might be possible, but we will have to split up the check functions some more, separate part to calculate.
Because of course we still only want to performDrawCards once here (for instance so you know all new drawn cards to highlight). Also want to avoid that we do multiple identical queries to check if inflation is active anywhere etc.
I will look into this later.

Reply:
Pushed another commit to split the calculation part of the party/poor bonus from the actually applying it part to draw extra.
So now we can reuse the calculation part in the DrawCardsTrait.
Since the Inflation active check is just a game state, it doesn't really matter we have some extra checks on it.